docs/document/docs/components/desktop/button.md

```diff
@@ -25,7 +25,8 @@ Here is a list of properties that can be used for modifying the component:
 | :--- | :--- | :--- | :--- | :--- |
 | className | string | ""  | Component class name |  |
 | id | string | ""  | Component id name |  |
-| text | string | ""  | Text to be displayed inside the button | |
+| text | string | ""  | Text to be displayed inside the button | If `content` is unspecified, the value of `text` will be displayed<br>In other cases, the `text` will be ignored |
+| content | string/HTMLElement | ""  | The DOM inside button | If a string with HTML is set, it will be automatically converted to HTML and displayed as it is |
```

Comment:
I missed one point to add security related description like Dialog content. Please arrange that.
http://kintone-ui-component.netlify.com/docs/en/components/desktop/dialog

> *1: [Security] Kintone UI Component does NOT sanitize this property value. It is the developer's responsibility to escape any user input when using this option so that XSS attacks would be prevented.